### PR TITLE
Add pathlib support for file operations

### PR DESCRIPTION
**Pull request by wsapia**

### CODE REVIEW

File: skrf/util.py [hunk 1]
```diff
@@ -282,13 +283,13 @@ def get_fid(file, *args, **kwargs):
     fid : file object
 
     """
-    if isinstance(file, str):
+    if isinstance(file, str) or isinstance(file, Path):
```
Comment (FranzForstmayr) [inline skrf/util.py:286]:
```suggestion
    if isinstance(file, (str, Path)):
```

**Comment by FranzForstmayr** [comment]
Great improvement, thank you!

**Comment by wsapia** [comment]
Not really sure why that last set of checks failed... do you see anything obvious?

**Comment by FranzForstmayr** [comment]
No idea either, I just reran the pipeline and it is green now.

**Comment by wsapia** [comment]
Ok, great, glad it worked this time.

**Review by jhillairet — APPROVED** [review]
Nice addition, thanks!

**Comment by jhillairet** [comment]
ok to be merged?

**Comment by wsapia** [comment]
Sure, go for it!

**Pull request merged**